sorc/ncep_post.fd/MDL2P.f

```diff
@@ -4082,13 +4083,18 @@ SUBROUTINE MDL2P(iostatusD3D)
          if(me==0)PRINT*,'CALLING SLR'
          egrid1=spval
          call calslr_roebber(TPRS,RHPRS,EGRID1)
+      !   call calslr_uutah(EGRID1)
```

Comment:
@JesseMeng-NOAA I wonder if you can use a flag to switch between two schemes. This flag can be read from itag as namelist. The default setting is Roebber scheme. The sample flag can be found in UPP code such as "gocart_on"

Reply:
OK I can do that.